### ISSUE TITLE
Multiple Ways to open the COnfigure Mech Dialog?

### ISSUE THREAD

**Issue by BlindGuyNW** [MegaMek/megamek#1335]
For ease of use and accessibility purposes, multiple ways should be provided to open the Configure Mech dialog. Perhaps the commo choice button could be repurposed or a separate "configure," button added? Havingonly one wayt to do something makes it harder to discover sometimes important corners of the UI.
